docker/Dockerfile_ubuntu_1604

```diff
@@ -0,0 +1,22 @@
+# Ubuntu 16.04 installed with the following libraries:
```

Comment:
Can we add commands in dockerfile that adds or mounts this repository to the container? After creating an image with this file, users might be confused about what to do next.

Reply:
My current idea is for users to either `git clone ...` the repo inside the docker container, or mount this repository to the container. This is not added with Dockerfile since I am not sure which solution best fits users' need.

In either case, we could probably assume that users know how to use docker, and users can choose the solution that best fits their development needs. What do you think?

Reply:
I think it's ok for me given that it is a demo and the PR pushes to the `dev` branch. When we want to further push to `master` branch I think it is better to add more detailed description in README about these information.